tests/onnx/test_onnx_helper.py

```diff
@@ -71,6 +71,22 @@ def test_pack_int4_to_uint8_unsigned_even_block():
     assert_array_equal(packed, expected)
 
 
+def test_pack_int4_to_uint8_unsigned_odd_block():
```

Comment:
Please add tests cease for all arguments, differ input dtype, differ bloack_size, signed or not

Reply:
We already have several tests:

test_pack_int4_to_uint8_unsigned_even_block
test_pack_int4_to_uint8_unsigned_odd_block
test_pack_int4_to_uint8_signed
test_pack_int4_to_uint8_multiple_blocks
test_pack_int4_to_uint8_raises_on_invalid_signed_dtype

These should cover everything.